### ISSUE TITLE
Unit error in Modelica.Media.Examples.SimpleLiquidWater

### ISSUE THREAD

**Issue by qlambert-pro** [modelica/ModelicaStandardLibrary#4099]
The following equation does not pass unit-checking:
```
medium2.p = 1e5 * time / 10;
```

`medium2.p` is expressed in `"Pa"` and `time` in `"s"`

**Comment by qlambert-pro** [comment]
`Modelica.Media.Examples.ReferenceAir.MoistAir` suffers from the same problem for the following equations:
```
  state1.p = 1.e5 * (1 + time);
  state1.T = 300 + 10 * 
  state2.p = 1.e5 * (1 + time / 2);
  state2.T = 340 - 20 * time;
```

**Comment by qlambert-pro** [comment]
`ModelicaStandardLibrary/issues/ModelicaTest.Media.TestOnly.MoistAir` suffers from the same problem for the following binding equation:

```
  SI.AbsolutePressure p = 2e5 - 1.5e5 * time;
```

**Comment by beutlich** [comment]
Resolved by #4103.
